addons/findAll/findAll.lua

```diff
@@ -48,6 +48,7 @@ res = require('resources')
 defaults = {}
 defaults.Track = ''
 defaults.Tracker = {}
+defaults.Tracker.Key_Item_Display = true
```

Comment:
Is the key item display related to the tracker? If not, I'd move the setting out of that table. Also please remove the underscores from settings names:

```lua
defaults.KeyItemDisplay = true
```